website/docs/streaming-platforms/nifi.md

```diff
@@ -0,0 +1,205 @@
+logging             aws-for-fluent-bit-rq2nc                                     1/1     Running   1 (43h ago)   2d
+logging             aws-for-fluent-bit-tnmtl                                     1/1     Running   1 (43h ago)   2d
+logging             aws-for-fluent-bit-zzhfc                                     1/1     Running   1 (43h ago)   2d
+nifi                nifi-0                                                       5/5     Running   0             41h
+nifi                nifi-1                                                       5/5     Running   0             41h
+nifi                nifi-2                                                       5/5     Running   0             41h
+nifi                nifi-registry-0                                              1/1     Running   0             41h
+nifi                nifi-zookeeper-0                                             1/1     Running   0             41h
+nifi                nifi-zookeeper-1                                             1/1     Running   0             41h
+nifi                nifi-zookeeper-2                                             1/1     Running   0             18h
+prometheus          prometheus-alertmanager-655fcb46df-2qh8h                     2/2     Running   2 (43h ago)   2d
+prometheus          prometheus-kube-state-metrics-549f6d74dd-wwhtr               1/1     Running   1 (43h ago)   2d
+prometheus          prometheus-node-exporter-5cpzk                               1/1     Running   0             19h
+prometheus          prometheus-node-exporter-8jhbk                               1/1     Running   1 (43h ago)   2d
+prometheus          prometheus-node-exporter-nbd42                               1/1     Running   1 (43h ago)   2d
+prometheus          prometheus-node-exporter-str6t                               1/1     Running   1 (43h ago)   2d
+prometheus          prometheus-node-exporter-zkf5s                               1/1     Running   1 (43h ago)   2d
+prometheus          prometheus-pushgateway-677c6fdd5-9tqkl                       1/1     Running   1 (43h ago)   2d
+prometheus          prometheus-server-7bf9cbb9cf-b2zgl                           2/2     Running   2 (43h ago)   2d
+vpa                 vpa-recommender-7c6bbb4f9b-rjhr7                             1/1     Running   1 (43h ago)   2d
+vpa                 vpa-updater-7975b9dc55-g6zf6                                 1/1     Running   1 (43h ago)   2d
+```
+
+#### Apache NiFi UI
+
+The Ray Dashboard can be opened at the following url "https://nifi.example.com/nifi"
+
+![Apache NiFi Login](img/nifi.png)
+
+Run the command below to retrieve NiFi user's password for login
+```
+aws secretsmanager get-secret-value --secret-id nifi_login_password --region <region> | jq '.SecretString' --raw-output
+```
+
+![Apache NiFi Canvas](img/nifi-canvas.png)
+
+
+### Examples
+
+Coming soon..
```

Comment:
Can we add a simple example now? Without example the blueprint does not make much sense.

Reply:
Example added